pinc/CharSuites.inc

```diff
@@ -178,3 +178,45 @@ foreach($charsuite_files as $charsuite_file)
 {
     include_once($charsuite_file);
 }
+
+function get_pickersets($charsuites)
```

Comment:
I also wonder if these should be static functions inside the `CharSuites` class to help indicate what they are getting the pickersets from:
```
// Given a character suite (by object or name, or an array of either)
// return the picker sets associated with them.
public static function get_pickersets($charsuites)
{
    if(!is_array($charsuites))
    {
        $charsuites = [ $charsuites ];
    }

    $pickersets = [];
    foreach($charsuites as $charsuite)
    {
        $charsuite = CharSuites::resolve($charsuites);
        $pickersets[] = $charsuite->pickerset;
    }
    return $pickersets;
}
```

Ditto `get_valid_codepoints()`.

Reply:
They will be in effect inside a project class which will have the same sort of effect.